cogctl/cli/upgrade_configuration.py

```diff
@@ -0,0 +1,22 @@
+import click
+
+
+@click.command(name="upgrade-configuration")
+@click.pass_obj
+def upgrade_configuration(state):
```

Comment:
Normally I agree with the explicit bit. But in this case it seems like something that would end up eliminating some edge cases if we just did it automatically. If we just automatically upgrade the config then we only have to support the new style config. But if we make it an option then we would need to support both the new style and old style config, which leaves more room for funky edge cases. But regardless, it's probably not that big of a deal right now since the config files really don't have that much in them.